.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,6 +3,11 @@ name: Bug report
 about: Create a report to help reproduce and fix the issue
 ---
 <!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable -->
+
+## Issue Summary
+
+<!-- Briefly explain your issue in a few plain sentences. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue" You may copy and paste the issue title here if it is suitable -->
```

Comment:
```suggestion
<!-- Briefly explain your issue in a few plain sentences, e.g. "X has Y issue" "When I do X, Y happens instead of Z" "X on Y map has Z issue". You may copy and paste the issue title here if it is suitable. -->
```

formatting nits